src/Apps/Search/Routes/Artists/SearchResultsArtists.tsx

```diff
@@ -66,7 +66,8 @@ export class SearchResultsArtistsRoute extends React.Component<
   }
 
   renderArtists() {
-    const { viewer } = this.props
+    const { viewer, location } = this.props
+    const { term } = get(location, l => l.query)
```

Comment:
Without adding a `{}` as a fallback (third param) destructuring like this will error out if there's no term.

Reply:
I think that's the default already? When adding that I get a type error on the `const {term}` part.

Reply:
This demonstrates it well: 

https://npm.runkit.com/ts-get

And then paste this in: 

```
var { get }  = require("ts-get")
const location = {}
const { term } = get(location, l => l.query)
console.log(term)
```

Reply:
Yea, the typing from `found` indicates `query` isn't optional.